tools/pfif-tools/app/settings.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# If we actually did anything that used the secret key we'd need to set it to
+# some constant value and find a way to secretly store it. However, pfif-tools
+# doesn't use it for anything, we just need to set it to something to make
+# Django happy.
+SECRET_KEY = os.urandom(30)
```

Comment:
Optional: If it isn't used for anything, maybe it can even be an empty string or a constant string like 'dummy'?

Reply:
Just to be safe in case we end up accidentally using it in the future. Added to the comment in the file.